src/components/templates/layouts/AuthLayout/AuthLayout.tsx

```diff
@@ -2,14 +2,12 @@ import React from 'react'
 import styled from 'styled-components'
 
 export interface AuthLayoutProps {
-  authLayout?: AuthLayoutType
+  authLayout?: string
   bgImage?: string
   className?: string
   style?: React.CSSProperties
 }
 
-export type AuthLayoutType = '' | 'center-full-bg' | 'full-page-left' | 'full-page-right'
```

Comment:
Why did you remove the Interface here?

Reply:
because it caused trouble with federation and I have to refactor that this week 👍